make install no longer works

This started happening sometime after May 23.

When I follow the instructions in install.md to build from source, when I get to the `make install` portion it fails with the following message:

```
# make install
GO111MODULE=on go build -mod=vendor "-buildmode=pie" -ldflags '-X main.gitCommit=e8a3064328c535ffecfa60e1e2611530e721153c ' -gcflags "" -tags "   " -o bin/skopeo ./cmd/skopeo
install -d -m 755 /usr/local/bin
install -m 755 bin/skopeo /usr/local/bin/skopeo
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-copy.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-copy.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-delete.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-delete.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-inspect.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-inspect.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-list-tags.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-list-tags.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-login.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-login.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-logout.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-logout.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-manifest-digest.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-manifest-digest.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-standalone-sign.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-standalone-sign.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-standalone-verify.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-standalone-verify.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo-sync.1.md  | go-md2man -in /dev/stdin -out docs/skopeo-sync.1
sed -e 's/\((skopeo.*\.md)\)//' -e 's/\[\(skopeo.*\)\]/\1/' docs/skopeo.1.md  | go-md2man -in /dev/stdin -out docs/skopeo.1
install -d -m 755 /usr/local/share/man/man1
install -m 644 docs/*.1 /usr/local/share/man/man1
install -d -m 755 completions/{bash,zsh,fish,powershell}
./bin/skopeo completion bash >| completions/bash/skopeo
/bin/sh: 1: cannot create completions/bash/skopeo: Directory nonexistent
make: *** [Makefile:157: completions] Error 2
root@5ed04190c63b:/opt/go/src/github.com/containers/skopeo# ls completions/
{bash,zsh,fish,powershell}
root@5ed04190c63b:/opt/go/src/github.com/containers/skopeo#
```

--- a/Makefile
+++ b/Makefile
@@ -153,7 +153,7 @@ docs-in-container:
 
 .PHONY: completions
 completions: bin/skopeo
-	install -d -m 755 completions/{bash,zsh,fish,powershell}
+	install -d -m 755 completions/bash completions/zsh completions/fish completions/powershell
 	./bin/skopeo completion bash >| completions/bash/skopeo
 	./bin/skopeo completion zsh >| completions/zsh/_skopeo
 	./bin/skopeo completion fish >| completions/fish/skopeo.fish

